fetch.bs

```diff
@@ -6714,6 +6753,435 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
 </ol>
 </div>
 
+<h3 id=deferred-fetch>Deferred fetching</h3>
+
+<p>Deferred fetching allows callers to request that a fetch is invoked at the latest possible
+moment, i.e., when a <a>fetch group</a> is <a for="fetch group">terminated</a>, or after a timeout.
+
+<p>The <dfn>deferred fetch task source</dfn> is a <a>task source</a> used to update the result of a
+deferred fetch. User agents must prioritize tasks in this <a>task source</a> before other task
+sources, specifically task sources that can result in running scripts such as the
+<a>DOM manipulation task source</a>, to reflect the most recent state of a
+<a method><code>fetchLater()</code></a> call before running any scripts that might depend on it.
+
+<div algorithm>
+<p>To <dfn>queue a deferred fetch</dfn> given a <a for=/>request</a> <var>request</var>, a null or
+{{DOMHighResTimeStamp}} <var>activateAfter</var>, and <var>onActivatedWithoutTermination</var>,
+which is an algorithm that takes no arguments:
+
+<ol>
+ <li><p><a>Populate request from client</a> given <var>request</var>.
+
+ <li><p>Set <var>request</var>'s <a for=request>service-workers mode</a> to "<code>none</code>".
+
+ <li><p>Set <var>request</var>'s <a for=request>keepalive</a> to true.
+
+ <li><p>Let <var>deferredRecord</var> be a new <a>deferred fetch record</a> whose
+ <a for="deferred fetch record">request</a> is <var>request</var>, and whose
+ <a for="deferred fetch record">notify invoked</a> is
+ <var>onActivatedWithoutTermination</var>.
+
+ <li><p><a for=list>Append</a> <var>deferredRecord</var> to <var>request</var>'s
+ <a for=request>client</a>'s <a>fetch group</a>'s <a for="fetch group">deferred fetch records</a>.
+
+ <li>
+  <p>If <var>activateAfter</var> is non-null, then run the following steps <a>in parallel</a>:
+
+  <ol>
+   <li>
+    <p>The user agent should wait until any of the following conditions is met:
+
+    <ul class=brief>
+     <li><p>At least <var>activateAfter</var> milliseconds have passed.
+
+     <li><p>The user agent has a reason to believe that it is about to lose the opportunity to
+     execute scripts, e.g., when the browser is moved to the background, or when
+     <var>request</var>'s <a for=request>window</a> is a {{Window}} whose
+     <a>associated document</a> had a "<code>hidden</code>" <a for=Document>visibility state</a> for
+     a long period of time.
+    </ul>
+
+   <li><p><a data-lt="process a deferred fetch">Process</a> <var>deferredRecord</var>.
+  </ol>
+ </li>
+
+ <li><p>Return <var>deferredRecord</var>.
+</ol>
+</div>
+
+<div algorithm>
+<p>To compute the <dfn>total request length</dfn> of a <a for=/>request</a> <var>request</var>:
+
+<ol>
+ <li><p>Let <var>totalRequestLength</var> be the <a for=string>length</a> of <var>request</var>'s
+ <a for=request>URL</a>, <a lt="URL serializer">serialized</a> with
+ [=URL serializer/exclude fragment=] set to true.
```

Comment:
I don't think this will get the correct formatting. It needs to use `<i><a>` iirc.